packages/atomic/src/components/result-template-components-v1/atomic-result-multi-text/atomic-result-multi-text.tsx

```diff
@@ -0,0 +1,162 @@
+
+  /**
+   * The result field which the component should use.
+   * This will look in the Result object first, and then in the Result.raw object for the fields.
+   * It is important to include the necessary field in the ResultList component.
+   */
+  @Prop() public field!: string;
+
+  /**
+   * If present, will attempt to split the field into multiple values using wherever this character is encountered.
+   */
+  @Prop() public delimitingCharacter?: string;
+
+  /**
+   * The maximum number of field values to display.
+   * If there are more values than the maximum that was set here, the last value displayed will be "# more...".
+   */
+  @Prop() public maximumValues = 3;
+
+  public initialize() {
+    this.breadcrumbManager = buildBreadcrumbManager(this.bindings.engine);
+  }
+
+  private get resultValues() {
+    const value = ResultTemplatesHelpers.getResultProperty(
+      this.result,
+      this.field
+    );
+
+    if (Array.isArray(value)) {
+      return value.map((v) => `${v}`);
+    }
+
+    if (typeof value !== 'string' || value.trim() === '') {
+      return [];
+    }
+
+    return this.delimitingCharacter
+      ? value.split(this.delimitingCharacter)
+      : [value];
+  }
+
+  private get facetSelectedValues() {
+    return this.breadcrumbManager.state.facetBreadcrumbs
+      .filter((facet) => facet.field === this.field)
+      .reduce(
+        (values, facet) => [
+          ...values,
+          ...facet.values.map(({value}) => value.value),
+        ],
+        [] as string[]
+      );
+  }
+
+  private getSortedValues() {
+    const allValues = this.resultValues;
+    const allValuesSet = new Set(this.resultValues);
+    const firstValues = this.facetSelectedValues.filter((value) =>
+      allValuesSet.has(value)
+    );
+    return Array.from(
+      allValues.reduce((set, value) => set.add(value), new Set(firstValues))
+    );
+  }
+
+  private renderValue(value: string) {
+    const label = getFieldValueCaption(this.field, value, this.bindings.i18n);
+    const kebabValue = titleToKebab(value);
+    return (
+      <li key={value} part="result-multi-text-value" class={listItemClasses}>
+        <slot name={`result-multi-text-value-${kebabValue}`}>{label}</slot>
+      </li>
+    );
+  }
+
+  private renderSeparator(beforeValue: string, afterValue: string) {
+    return (
+      <li
+        role="separator"
+        part="result-multi-text-separator"
+        key={`${beforeValue}~${afterValue}`}
+        class={`separator ${listItemClasses}`}
+      ></li>
+    );
+  }
+
+  private renderMoreLabel(value: number) {
+    return (
+      <li key="more-field-values" class={listItemClasses}>
+        {this.bindings.i18n.t('n-more', {value})}
+      </li>
+    );
+  }
+
+  private renderListItems() {
+    const sortedValues = this.getSortedValues();
+    const shouldDisplayLabel = sortedValues.length > this.maximumValues;
+    const numberOfValuesToDisplay = shouldDisplayLabel
+      ? this.maximumValues - 1
```

Comment:
Following on the earlier comment, I think this `-1` should be removed, leaving it to devs to set the value.

Reply:
The `-1` here can be seen as replacing the last value with the `{{value}} more...` message.

To make the size occupied more predictable, I prefer to keep this behaviour.